backend/backends/statsdaemon/statsdaemon.go

```diff
@@ -119,19 +120,24 @@ func (client *client) SendMetrics(metrics *types.MetricMap) error {
 	return nil
 }
 
+// SendEvent discards events.
```

Comment:
I don't think it should discard the event, the idea with this backend is that the front rows of statsd server will send to the statsd master which will have the backend logics. So it should send the event to the master.